### PR TITLE
[Fix] War room outputs ^^^ - JS

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/19812

## Description
Added spaces between item cells in `tableToMarkdown` to fix `^^^` chars showing when indicator values appear in the table cells.

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests - (`VerifyHumanReadableFormat`)
- [x] Code Review

## Additional changes
Describe additional changes done, for example adding a function to common server.

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)
- [ ] [CHANGELOG](link)

### CODE REVIEW

**Comment by ronykoz** [comment]
@DeanArbel is this ready for a review?

**Comment by ronykoz** [comment]
@yaakovi this build doesn't recognise the RN for some reason, can you take a look at it please?

File: Scripts/script-CommonServer_CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## [Unreleased]
+
+
+## [19.8.0] - 2019-08-06
```
Comment (ronykoz) [inline Scripts/script-CommonServer_CHANGELOG.md]: I think it should be in the Unrealeased section

Comment (DeanArbel) [reply Scripts/script-CommonServer_CHANGELOG.md]: you're right 👍

**Comment by kirbles19** [comment]
@DeanArbel Done.

**Pull request merged**